package.json

```diff
@@ -191,8 +192,8 @@
     "prebuild-server": "mkdirp build",
     "build-server": "npm run -s env -- webpack --display-error-details --config webpack.config.node.js",
     "build-client": "npm run -s env -- webpack --display-error-details --config webpack.config.js",
-    "build-client-if-prod": "node -e \"process.env.CALYPSO_ENV === 'production' && process.exit(1)\" || ( npm run build-client && npm run minify )",
-    "build-client-if-desktop": "node -e \"process.env.CALYPSO_ENV === 'desktop' && process.exit(1)\" || ( npm run build-client && npm run minify )",
+    "build-client-if-prod": "node -e \"process.env.CALYPSO_ENV === 'production' && process.exit(1)\" || npm run build-client",
+    "build-client-if-desktop": "node -e \"process.env.CALYPSO_ENV === 'desktop' && process.exit(1)\" || npm run build-client",
```

Comment:
Are these `build-client-if-*` scripts still needed? Their purpose seems to be only to run `npm run minify` conditionally for some environments. Can they be merged into one `build-client` now?

Reply:
we actually only invoke `build-client` when in production or desktop env.  build-client is a script that runs webpack on the client folder and outputs files to the public directory.

There are two configurations:
1. `production|desktop`: the server expects there to be js bundles already created in the public directory (build-client should run)
2. `development`: we never actually build client bundles to the filesystem.  the server during boot will [realize its development mode](https://github.com/Automattic/wp-calypso/blob/master/server/boot/index.js#L38) and programmatically trigger an in-memory build.

Reply:
>  Can they be merged into one build-client now?

so to answer the question: we can improve how we decide when to `build-client`, but I'd prefer to leave that for a later pr

Reply:
Thanks for explanation! I only now realized that this `build-if-*` conditional skips the `build-client` step when running `npm start` in development mode -- the client JS is served by the Webpack dev server in that case.